parsers/CA_QC.py

```diff
@@ -37,7 +35,9 @@ def fetch_production(
     now = datetime.now(tz=TIMEZONE)
     for elem in data:
         values = elem["valeurs"]
-        timestamp = arrow.get(elem["date"], tzinfo=TIMEZONE).datetime
+        timestamp = datetime.strptime(elem["date"], "%Y-%m-%dT%H:%M:%S").replace(
+            tzinfo=TIMEZONE
+        )
         # The datasource returns future timestamps or recent with a 0.0 value, so we ignore them.
```

Comment:
```suggestion
        if isinstance(elem["date"], str):
            timestamp = datetime.fromisoformat(elem["date"]).replace(tzinfo=TIMEZONE)
```
We should make sure the date field is a string before we use it and we should stick with fromisoformat whenever possible.